pyscriptjs/tests/integration/support.py

```diff
@@ -126,6 +127,12 @@ def init_page(self, page):
         page.on("console", self._on_console)
         page.on("pageerror", self._on_pageerror)
 
+    def disable_cors_headers(self):
+        if self.dev_server is None:
+            self.router.use_cors = False
+        else:
+            self.dev_server.disable_cors()
+
```

Comment:
see my comment above. If it's called `disable_cors_headers()` here, it should also be called `disable_cors_headers` in `dev_server`.
And probably in `router`, it should be called `use_cors_headers`.

Reply:
I am using `enable_cors_headers` to maintain consistency...
Or should I change it to `use_cors_headers` for `router`?